src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1461,7 +1461,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Predefined type '{0}' is not defined or imported</value>
   </data>
   <data name="ERR_PredefinedValueTupleTypeNotFound" xml:space="preserve">
-    <value>Predefined type '{0}' is not defined or imported</value>
+    <value>Predefined type '{0}' is not defined or imported, or is ambiguous (imported twice)</value>
```

Comment:
Sure. I'll adjust message.
I don't plan to update the baseline of every test. Let me know if that's a concern.